pkgs/applications/virtualization/hyperkit/dtrace.h

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
Where does it come from?

Reply:
Urgh yeah I forgot about this. This is from the output of https://github.com/moby/hyperkit/blob/master/Makefile#L154 - we don't have dtrace packaged so I can't really include it as a build step. The problem is that this might be platform specific, though it seems to work across the platforms I've tried.

Reply:
@copumpkin you probably know most about what it'd take to package dtrace. Do you think it'd be possible?

Reply:
If it not possible, it would be at least helpful to add a comment how it was created.

Reply:
I'd start by including this with a comment about where it came from for now. I do think we can package it (we sort of have a skeleton package for it) but it's a bigger story and not a crucial dependency.